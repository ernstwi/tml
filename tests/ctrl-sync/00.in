init a

a >> b
b -> c
    sync { foo }
