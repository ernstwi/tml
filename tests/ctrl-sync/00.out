Parse error at FILE "tests/ctrl-sync/00.in" 5:16: Unexpected '}'. Expected '?' or '!'
